modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -88,27 +116,93 @@ data:
         basicAuth:
           username:
             name: remoteWriteAuth
```

Comment:
we should mention what's remoteWriteAuth here.
FYI, for secret, the secret name is improper, since
```
The Secret "remoteWriteAuth" is invalid: metadata.name: Invalid value: "remoteWriteAuth": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Reply:
- changed name from `remoteWriteAuth` to `rw-basic-auth`
- Added a sentence saying `It also assumes that you have already created a `basicAuth` secret with the name `rw-basic-auth` in the `openshift-monitoring` namespace.`